src/components/cylc/Task.vue

```diff
@@ -178,6 +232,13 @@ export default {
       return {
         'stroke-dashoffset': `calc(157 - (157 * ${this.progress / 100} + ))`
       }
+    },
+    classes: function () {
+      var classes = [this.status]
```

Comment:
s/var/const

Reply:
Dammit, I'm really going to have to do some reading!

Reply:
I think `var` is considered deprecated (or even a risk, because... I think JS moves the var variable declarations to the top of the code, which may cause some weird bugs). If the variable doesn't change, `const`. If it needs to change (e.g. a counter), then `let`. I acquired the habit to always go with `const`, and change to `let` if necessary (used to do the same with Java's `final`)

Reply:
In lines 237-8 the list `classes` may get extended (`classes.push`) so should this be `let`?

Reply:
Sorry, I should have said "if the variable is not re-assigned". `const` creates a mutable variable.

You are correct that `classes` is changed, but only the internal structure, not the variable pointer. So still a `const`. Sorry for the confusion.

Reply:
Aaah, now it makes sense!